src/main/java/lt/vu/trip/entity/TripParticipation.java

```diff
@@ -21,6 +22,7 @@
 
 	@NotNull
 	@ManyToOne
+	@JsonBackReference
```

Comment:
Avoiding recursion when getting JSON of trip. (Trip has TripParticipations, which Trip)